var/spack/repos/builtin/packages/python/package.py

```diff
@@ -452,6 +452,13 @@ def flag_handler(self, name, flags):
             if name == 'cflags':
                 flags.append('-fwrapv')
 
+        # Fix for following issues for python with aocc%3.2.0:
+        # https://github.com/spack/spack/issues/29115
+        # https://github.com/spack/spack/issues/28708
+        if self.spec.satisfies('%aocc@3.2.0', strict=True):
```

Comment:
Does this bug not affect AOCC 3.2.1+?

Reply:
No, This issue is fixed in upcoming aocc compiler releases.